math/geometry/distancebetweenpoints.go

```diff
@@ -0,0 +1,53 @@
+// distancebetweenpoints.go
+// Find distance between two points
+// Details:
+// Given coordinates of 2 points on a cartesian plane, find the distance between them rounded up to nearest integer.
+// Link: https://practice.geeksforgeeks.org/problems/distance-between-2-points3200/1?page=1&category[]=geometric
+
+// author(s) [Chetan Patil](https://github.com/Chetan07j)
```

Comment:
```suggestion
// distancebetweenpoints.go
// Find Euclidean distance between two points
// author(s) [Chetan Patil](https://github.com/Chetan07j)
```

The link referred doesn't apply anymore since the algorithm is widely different from the initial one you proposed.

I also recommend changing the file name to `distance.go` or `euclideandistance.go`. I prefer the former.

Reply:
done